contrib/docker/Dockerfile.fpm-alpine

```diff
@@ -0,0 +1,94 @@
+FROM php:8.1-fpm-alpine AS app_php
+
```

Comment:
Since it's a new script this is a great time to fix the user id problems that plague administration for folks.